quantecon/markov/approximation.py

```diff
@@ -368,6 +366,39 @@ def discrete_var(A,
            [ 0.15422567, -0.03232648],
            [ 0.15422567, -0.03232648],
            [ 0.19278209, -0.03232648]])
+
+    The simulation below uses the same parameters with :math:`{u_t}`
+    drawn from a multivariate t-distribution
+
+    >>> mc = discrete_var(A, C, grid_sizes, 
+    ...               rv=scipy.stats.multivariate_t(shape=np.identity(2)), 
```

Comment:
Thanks @HumphreyYang, but it is the covariance matrix that must be the identity matrix.

Reply:
Many thanks @oyamad. Sorry I forgot the constraint on the covariance matrix.

Do you think we should add these lines below into the docstring as well, in addition to my update to emphasize our constraint on the distribution?

```python
>>> df = 100
>>> Sigma = np.diag(np.tile((df-2)/df, 2))
>>> t_dis = scipy.stats.multivariate_t(shape=Sigma, df=df).rvs(10_000_000)
>>> np.round(np.cov(t_dis.T), 2)
array([[1., 0.],
       [0., 1.]])
```